### ISSUE TITLE
Report error when sizes="" is used on non-icon <link>

### ISSUE THREAD

**Issue by zcorpan** [validator/validator#399]
> The sizes attribute is used with the icon link type. The attribute must not be specified on link elements that do not have a rel attribute that specifies the icon keyword.

https://html.spec.whatwg.org/multipage/semantics.html#the-link-element:attr-link-sizes-2

```html
<!DOCTYPE html>
<html>
<head>
<title>Test</title>
<link href=x rel=stylesheet sizes="16x16">
</head>
<body>
<p></p>
</body>
</html>
```
> Document checking completed. No errors or warnings to show.

There should be an error
